### ISSUE TITLE
need for spellflag_targ_no_static

### ISSUE THREAD

**Issue by xMirroRx** [Sphereserver/Source-X#1291]
spellflag_targ_no_static..
can a flag be added to prevent targeting static items? i'm thinking of adding this to spells that don't need to target the ground.

![2](https://github.com/user-attachments/assets/ec90a618-1197-4730-9749-1a5e07f07b4d)

**Comment by cbnolok** [comment]
Doesn't spellflag_targ_char, spellflag_targ_item or spellflag_targ_obj suit you?

**Comment by GladieUO** [comment]
Since we are talking about spellflags, I would also prefer something like, spellflag_notarg_item, since if you use spellflag_area, it goes through chars and items.

**Comment by xMirroRx** [comment]
No, none of them work completely. Also, the rejection request from Flags is not triggered with @Abort on Magery.

I can suggest the following as a definitive solution to this situation.

It should be activated before @SpellCast and @Start.

New Trigger @SpellSelectTarget // This trigger will be fired when the Player targets something with a spell.

If it targets the place, LOCAL.P will carry the location information of the place.
ARNG1 Spell Number.
UID Target owner.
SRC Target.
etc.

In this way, we can control exactly how we want which spell to be cast before it is cast.
